### PR TITLE
[gawk] Update documentation

### PR DESCRIPTION
**Pull request by gavindidrichsen**
Signed-off-by: Gavin Didrichsen <gavin.didrichsen@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,4 +12,52 @@ Binary package
 
 ## Usage
 
-*TODO: Add instructions for usage*
+```
+$ hab pkg install core/gawk
```
Comment (skpaterson) [inline README.md]: Can't this just be one command e.g. 

```
[1][default:/src:0]# hab pkg install core/gawk --binlink
» Installing core/gawk
☁ Determining latest version of core/gawk in the 'stable' channel
→ Using core/gawk/5.0.1/20200305233704
★ Install of core/gawk/5.0.1/20200305233704 complete with 0 new packages installed.
» Binlinking awk from core/gawk/5.0.1/20200305233704 into /bin
★ Binlinked awk from core/gawk/5.0.1/20200305233704 to /bin/awk
» Binlinking gawk from core/gawk/5.0.1/20200305233704 into /bin
★ Binlinked gawk from core/gawk/5.0.1/20200305233704 to /bin/gawk
» Binlinking gawk-5.0.1 from core/gawk/5.0.1/20200305233704 into /bin
★ Binlinked gawk-5.0.1 from core/gawk/5.0.1/20200305233704 to /bin/gawk-5.0.1
[2][default:/src:0]# gawk --help
```

Comment (gavindidrichsen) [reply README.md]: @skpaterson:  Updated and squashed the commits.

File: README.md [hunk 2]
```diff
@@ -12,4 +12,51 @@ Binary package
 
 ## Usage
 
-*TODO: Add instructions for usage*
+```bash
+$ hab pkg install core/gawk --binlink
```
Comment (skpaterson) [inline README.md]: I'd suggest it's worthwhile leaving the hab printout here - otherwise all you see is the `--help` output.

Comment (gavindidrichsen) [reply README.md]: Amended differently but after pattern of flex example.

**Review by skpaterson — APPROVED** [review]
Thanks @gavindidrichsen

**Pull request merged**